### DISCUSSION TITLE
v0.2.0

### DISCUSSION THREAD

**Discussion by AzulGarza** [Nixtla/nixtla#264] [Announcements]
### Features
* feat: SDK support Fern Regeneration -- March 22, 2024 in https://github.com/Nixtla/nixtla/pull/261
* feat: replace finetune with fewshot  in https://github.com/Nixtla/nixtla/pull/262
* feat: use new endpoints and rename models in https://github.com/Nixtla/nixtla/pull/263

### Documentation
* docs: add azure automl experiment  in https://github.com/Nixtla/nixtla/pull/246
* docs: add finetuning loss function tutorial and link it to finetuning tutorial in https://github.com/Nixtla/nixtla/pull/239
* docs: tutorial on long-horizon forecasting in https://github.com/Nixtla/nixtla/pull/249
* docs: add new faqs section  in https://github.com/Nixtla/nixtla/pull/250
* docs: add new sections to documentation  in https://github.com/Nixtla/nixtla/pull/255
* docs: add amazon chronos benchmark in https://github.com/Nixtla/nixtla/pull/257
* docs: update README.md in https://github.com/Nixtla/nixtla/pull/258

### Fixes
* chore: add pre-commit in https://github.com/Nixtla/nixtla/pull/252
* chore: collapse experiment results in pr  in https://github.com/Nixtla/nixtla/pull/247
* fix: all transform_inputs to handle cases where index column is set as time_col  in https://github.com/Nixtla/nixtla/pull/254
* hotfix: ignore precommit fern in https://github.com/Nixtla/nixtla/pull/260

## New Contributors
* @marcopeix made their first contribution in https://github.com/Nixtla/nixtla/pull/239
* @MMenchero made their first contribution in https://github.com/Nixtla/nixtla/pull/250
* @Yibei990826 made their first contribution in https://github.com/Nixtla/nixtla/pull/254
* @mergenthaler made their first contribution in https://github.com/Nixtla/nixtla/pull/258

**Full Changelog**: https://github.com/Nixtla/nixtla/compare/v0.1.21...v0.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Nixtla/nixtla/releases/tag/v0.2.0'>v0.2.0</a>.</em>

**Comment by mergenthaler** [comment]
Thanks @AzulGarza and team. This is a great release and amazing work.
